demo/src/screens/componentScreens/TimelineScreen.tsx

```diff
@@ -0,0 +1,139 @@
+import React, {useCallback, useRef, useState} from 'react';
+import {StyleSheet, ScrollView} from 'react-native';
+import {Assets, Colors, Timeline, View, Card, Text, Button} from 'react-native-ui-lib';
+
+const contents = [
+  'CURRENT (default) state with dashed line.\nAligned to title',
+  'SUCCESS state with label.',
+  'ERROR state with icon.',
+  'Custom color with icon and outline.\nAligned to title',
+  'NEXT state with outline.',
+  'NEXT state with circle point and entry point.'
+];
+
+const TimelineScreen = () => {
+  const [targetIndex, setTargetIndex] = useState(0);
+  const [expand, setExpand] = useState(false);
+  const target = useRef();
+
+  const onPress = useCallback(() => {
+    setTargetIndex(targetIndex === 0 ? 1 : 0);
+  }, [targetIndex]);
+
+  const onPressExpand = useCallback(() => {
+    setExpand(!expand);
+  }, [expand]);
+
+  const renderContent = (index = 0, targetRef?: any) => {
+    return (
+      <Card padding-page>
+        <Text text70BO ref={targetRef}>
+          Step {index + 1}
+        </Text>
+        <View marginT-5 padding-8 bg-grey70 br30>
+          <Text>{contents[index]}</Text>
+          <Button marginT-10 size={'small'} label={'Expand'} onPress={onPressExpand}/>
+          {expand && <View style={{height: 100, margin: 10, backgroundColor: 'red'}}/>}
+        </View>
+      </Card>
+    );
+  };
+
+  return (
+    <>
+      <Text h1 $textDefault margin-20>
+        Timeline
+      </Text>
+      <Button marginT-10 size={'small'} label={'Change target'} onPress={onPress}/>
```

Comment:
Can maybe change the UI of this button, it's not clear enough what it does. 
Maybe a toggle between "anchor point mode" and "default mode" or something similar..

Reply:
Sure